stdlib/asyncio/threads.pyi

```diff
@@ -1,7 +1,9 @@
 import sys
-from typing import Any, Callable, TypeVar
+from typing import Callable, TypeVar
+from typing_extensions import ParamSpec
 
-_T = TypeVar("_T")
+_P = ParamSpec("_P")
+_R = TypeVar("_R")
 
 if sys.version_info >= (3, 9):
-    async def to_thread(__func: Callable[..., _T], *args: Any, **kwargs: Any) -> _T: ...
+    async def to_thread(func: Callable[_P, _R], /, *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
```

Comment:
```suggestion
    async def to_thread(__func: Callable[_P, _R], /, *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
```
according to stubtest it's positional-only

Reply:
Ah stubtest doesn't support positional only `/`?

Reply:
Several type-checkers don't support it at all :/ https://github.com/python/typeshed/issues/4972